client/src/components/user-admin/UserManagement.js

```diff
@@ -44,64 +55,77 @@ const UserManagement = ({ users, setUserToEdit }) => {
   return (
     <div className="container--usermanagement">
       <div>
-        <h3>User Management</h3>
-        <input
+        <Typography variant='h3' sx={h3sx}>User Management</Typography>
+        
+        <div className="tab-buttons">
+          <ButtonGroup>
+            <div>
+              <Button
+                type="button"
+                className={
+                  searchResultType === 'name'
+                    ? 'select-button selected'
+                    : 'select-button'
+                }
+                variant='secondary'
+                onClick={buttonSwap}
+                disabled={searchResultType === 'name'}
+              >
+                Results by Name
+              </Button>
+            </div>
+            <div>
+              <Button
+                type="button"
+                className={
+                  searchResultType === 'email'
+                    ? 'select-button selected'
+                    : 'select-button'
+                }
+                variant='secondary'
+                onClick={buttonSwap}
+                disabled={searchResultType === 'email'}
+              >
+                Results by Email
+              </Button>
+            </div>
+          </ButtonGroup>
+        </div>
+        <TextField
           type="text"
           placeholder="Search by name and email..."
           value={searchTerm}
           onChange={handleChange}
         />
-        <div className="tab-buttons">
+        <Box>
```

Comment:
Feel free to steal from other components in the project already:

This section with rendered users is similar to the results from the `Project Management` page:

```jsx
        <Box sx={{ bgcolor: '#F5F5F5', my: 3 }}>
            <Box sx={{ py: 2, px: 4, ...childrenBoxSx }}>
                {children} {/* This is where searchRuslts.map() would go /*}
            </Box>
        </Box>
```